Just/Just.swift

```diff
@@ -901,7 +901,20 @@ public final class HTTP: NSObject, NSURLSessionDelegate, JustAdaptor {
                         task.resume()
                     }
                     if isSync {
-                        dispatch_semaphore_wait(semaphore, DISPATCH_TIME_FOREVER)
+                        // Express one second in nanoseconds.
+                        let NSEC: Int64 = 1000000000
+
+                        if let tint = timeout {
```

Comment:
`tint` doesn't make much sense. Perhaps mask the old variable aka `if let timeout = timeout`?